### ISSUE TITLE
Notification email failed: timed out!

### ISSUE THREAD

**Issue by cbrtumkur** [ccrisan/motioneyeos#2195]
### Preliminary Docs

I confirm that I have read the [CONTRIBUTING](https://github.com/ccrisan/motioneyeos/blob/master/.github/CONTRIBUTING.md) guide before opening this issue.

I confirm that I have read the [FAQ](https://github.com/ccrisan/motioneyeos/wiki/FAQ) before opening this issue.

### motionEyeOS Version

I am running motionEyeOS version: (insert your version here, e.g. 20180314).

### Board Model

I am using the following board/model: (insert your board model here, e.g. Raspberry PI 3B+).

### Camera

I am using the following type of camera: (choose from V4L2, MMAL, Network Camera, Fast Network Camera and Simple MJPEG Camera).

My camera model is: (insert your camera model, e.g. Logitech C290).

### Network Connection

My motionEyeOS unit is connected to the network via: (choose your network connection, e.g. WiFi, Ethernet).

### Peripherals

I am using the following peripherals that I consider relevant to this issue:

 * (e.g. USB WiFi adapter)
 * (e.g. External Hard Disk)

### Log Files

I consider the following log files relevant to this issue:

 * (attach e.g. motioneye.log)
 * (attach e.g. boot.log)

**Comment by Vilquid** [comment]
### Preliminary Docs
I confirm that I have read the [CONTRIBUTING](https://github.com/ccrisan/motioneyeos/blob/master/.github/CONTRIBUTING.md) guide before opening this issue.

I confirm that I have read the [FAQ](https://github.com/ccrisan/motioneyeos/wiki/FAQ) before opening this issue.

### motionEyeOS Version
I am running motionEyeOS version: 20200606.

### Board Model
I am using the following board/model: Raspberry PI 3B+.

### Camera
I am using the following type of camera: V4L2

My camera model is: [Aukey Webcam Full HD 1080p](https://www.ldlc.com/fiche/PB00394614.html)

### Network Connection
My motionEyeOS unit is connected to the network via: WiFi.

### Peripherals
I am not using others peripherals.




### Log Files
I consider the following log files relevant to this issue:

[motioneye.log](https://github.com/ccrisan/motioneyeos/files/8008204/motioneye.log)
[boot.log](https://github.com/ccrisan/motioneyeos/files/8008200/boot.log)
![image](https://user-images.githubusercontent.com/60876471/152647738-dcac2ad9-36a8-46fc-8e43-19b4e286ef13.png)

**Comment by starbasessd** [comment]
ERROR: failed to send mail: (535, '5.7.8 Username and Password not accepted. Learn more at\n5.7.8  https://support.google.com/mail/?p=BadCredentials v18sm1790178wrm.105 - gsmtp') indicated you don't have correct username or correct password for the account being used. The correct setup is:
![tempsnip](https://user-images.githubusercontent.com/20134429/152647481-7825deb0-ab0b-4a57-bf0d-17c513506d46.png)
I use the same email address: me@gmail.com for Email Address, SMTP Account, and From Address. For gmail, I use SMTP Port 587, and Use TLS ON. (Attached Pictures Time span is up to you, I never use more than 2-5 seconds...

**Comment by Vilquid** [comment]
So, the correct completion is ? :
Email addresses : AAA@gmail.com
SMTP Server : smtp.gmail.com
SMTP Port : 587
SMTP Account : BBB@gmail.com
SMTP Password : password_of_BBB@gmail.com
From Address : CCC@gmail.com
Use TLS : ON
Attached Pictures Time Span : 3

**Comment by starbasessd** [comment]
That should work, but the from & SMTP names are usually the same...

**Comment by Vilquid** [comment]
I have this error message :
![image](https://user-images.githubusercontent.com/60876471/152648325-05a6eccf-6a78-45b7-9e88-57974c1d39a6.png)
I don't know how it can help me because password_of_BBB@gmail.com and BBB@gmail.com are correct.

Email addresses : [AAA@gmail.com](mailto:AAA@gmail.com)
SMTP Server : smtp.gmail.com
SMTP Port : 587
SMTP Account : [BBB@gmail.com](mailto:BBB@gmail.com)
SMTP Password : [password_of_BBB@gmail.com](mailto:password_of_BBB@gmail.com)
From Address : [BBB@gmail.com](mailto:CCC@gmail.com)

**Comment by starbasessd** [comment]
There are issues with some 'special' characters in the password in motionEye & motionEyeOS. And you don't tack on the "@gmail.com" to the password, I hope. 
If you  are using special characters, (depending on which specific ones) you may be able to edit the /data/etc/camera-X.conf file and place it directly in there.

**Comment by Vilquid** [comment]
Yes I don’t tack @gmail.com. The problem is I don’t use special characters in my password. I gonna try to place my password in the file.

**Comment by starbasessd** [comment]
Also, please try dev20201026.

**Comment by Vilquid** [comment]
I tried to modify the /data/etc/camera-X.conf file but there is no file named camera-X.conf on the micro sd card. I also tried the dev20201026 version but Wifi doesn't work.

**Comment by starbasessd** [comment]
The same wpa_supplicant.conf file that works on 20200606 works on dev20201026.
You need to edit it while it's up and running (SSH or Console) or use a Linux to mount and edit the file.
